### ISSUE TITLE
Problem with recalculating the pipe after a dependency change with an intersection curve 

### ISSUE THREAD

**Issue by Fluppi86** [realthunder/FreeCAD_assembly3#1280]
Hi
I have now constructed a larger part at Freecad, I have a problem with the "Pipe" function.
I am no longer sure whether this is done at all.
Basically I am quite new to Freecad. I put my sketches on DatumPlane and try to "feed" the others with a couple of sketches. Here it is still quite confusing and I have tried a lot or always adapted and changed, so that you could do it with more structure.

The problem is that I have constructed a pipe with start profile, path and end profile. Once additive and subtrative.

I made sketches and used reference points from another sketch for the total size. The path, however, I made from a section curve. That is, 2 sketches extruded and then made a section curve from it.

![](https://i.postimg.cc/g0JTZG07/extrud.png)
![](https://i.postimg.cc/XJ9cZP9h/schnittkurve.png)

The whole thing worked, but when I change the total size with the first sketch, the pipe is not updated. I tried a little bit and I have the impression that I have to recreate the sketches that have a dependency on the intersection curve so that I can construct the pipe again.
I tried to reset the dependencies but that didn't help. 

![](https://i.postimg.cc/SxqsqvKR/Bildschirmfoto-vom-2022-10-17-21-16-19.png)
This is how it should be. But is only shown as a preview.

![](https://i.postimg.cc/PfgJVxsm/Bildschirmfoto-vom-2022-10-17-21-17-09.png)
This is the case when I influence the dependency with the first sketch and the pipe is not updated.

![](https://i.postimg.cc/RVv505wk/Bildschirmfoto-vom-2022-10-17-19-50-06.png)
This was the first pipe with a further processing in which I then noticed that I have to be 1mm shorter.
All sketches have changed, the preview of the pipe is also correct but is not implemented.

In one of the 4 profile sketches there was an error message "Wire not Closed" but the sketch was fully constrained. Then I created the sketch again in the same way, i.e. the dependency of the intersection curve and then the profile all around. When I then constructed the pipe, it worked.

Basically, the error was solved by creating the sketches again with the dependency on the section curve.
Did I make mistakes that I don't see?
mfg Fluppi :)
